utils/update_checkout/update-checkout-config.json

```diff
@@ -224,6 +228,7 @@
                 "swift": "release/5.10",
                 "cmark": "release/5.10",
                 "llbuild": "release/5.10",
+                "swift-toolchain-sqlite": "release/5.10",
```

Comment:
Will we back-fill the release?

Reply:
I created the branches for 6.0 and 5.10 on the repo, not sure if there's more to do than that.